src/app/features/test-records/views/amended-test-record/amended-test-record.component.html

```diff
@@ -3,14 +3,13 @@
     <div class="govuk-notification-banner__header">
       <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">Important</h2>
     </div>
+
     <div class="govuk-notification-banner__content">
       <p class="govuk-notification-banner__heading">Amended Test Record - created at {{ testResult.createdAt | date: 'dd/MM/yyyy HH:mm' }}</p>
     </div>
   </div>
 
   <app-base-test-record [edit]="(edit$ | async) || false" [testResult]="testResult"></app-base-test-record>
```

Comment:
Can we make edit false  here since we can't edit an amended test record ? and we can get rid of `edit$` in the `amended-test-record.component.ts` as well since we don't need it here.

Reply:
Good spot. Suggestion applied.